package.json

```diff
@@ -50,12 +50,18 @@
     "wsrun": "^5.2.0"
   },
   "resolutions": {
```

Comment:
What is `"resolutions"` used for? Which tool needs it?

`<rant>`
Don't get me started on `package.json`, it was already doing too many things (dependencies, tasks, versions, labels, publishing to npm, etc., etc.) but is now being overloaded by various tools with yet more configuration.  Good luck!
`</rant>`

Reply:
`resolutions` allows you to force the versions of indirect dependencies without upgrading or forking the intermediate dependency.